.github/workflows/nightly-pax-test-mgmn.yaml

```diff
@@ -112,9 +118,9 @@ jobs:
       FILE_ISSUE: true
 
   finalize:
-    if: "!cancelled()"
+    if: ${{ !cancelled() }}
     needs: [metadata, run-jobs]
     uses: ./.github/workflows/_finalize.yaml
     with:
-      PUBLISH_BADGE: ${{ needs.metadata.outputs.PUBLISH }}
+      PUBLISH_BADGE: ${{ needs.metadata.outputs.PUBLISH == 'true' }}
```

Comment:
cause the type for the input is `bool` and the output is a string so need to cast to prevent 
```
The template is not valid. .github/workflows/nightly-pax-test-mgmn.yaml (Line: 119, Col: 22): Unexpected value 'true'
```

Reply:
I see. I was just curious since the docs appear to suggest that this is treated as a string

https://docs.github.com/en/actions/learn-github-actions/expressions#about-expressions

Reply:
Yeah I think `needs.metadata.outputs.PUBLISH` is string `'true'` but it doesn't get automatically converted. https://github.com/NVIDIA/JAX-Toolbox/blob/main/.github/workflows/_finalize.yaml#L7